neo4j/test-integration/dbconn_test.go

```diff
@@ -236,7 +234,7 @@ func TestConnectionConformance(ot *testing.T) {
 		{
 			name: "Next passed the summary",
 			fun: func(t *testing.T, c db.Connection) {
-				s, err := boltConn.Run(db.Command{Cypher: "RETURN datetime()"}, db.TxConfig{Mode: db.ReadMode})
+				s, err := boltConn.Run(db.Command{Cypher: "RETURN 42"}, db.TxConfig{Mode: db.ReadMode})
```

Comment:
Why did this test change?

Reply:
because the driver would send a UTC DateTime structure and the server would not understand it (yet).
Since that test does not really need any DateTime to make its point, I decided to switch to a simpler value.